Assets/MixedRealityToolkit/Inspectors/MixedRealityToolkitFacadeHandler.cs

```diff
@@ -25,20 +25,19 @@ static MixedRealityToolkitFacadeHandler()
 
         private static void OnPlayModeStateChanged(PlayModeStateChange obj)
         {
-            if (!MixedRealityToolkit.IsInitialized)
```

Comment:
Curious, why did we remove this segment here?

Reply:
It's already guaranteed to get called at least once by later methods so it was overkill.